transaction-status/src/lib.rs

```diff
@@ -320,17 +328,26 @@ pub struct UiTransactionStatusMeta {
     pub fee: u64,
     pub pre_balances: Vec<u64>,
     pub post_balances: Vec<u64>,
-    pub inner_instructions: Option<Vec<UiInnerInstructions>>,
-    pub log_messages: Option<Vec<String>>,
-    pub pre_token_balances: Option<Vec<UiTransactionTokenBalance>>,
-    pub post_token_balances: Option<Vec<UiTransactionTokenBalance>>,
-    pub rewards: Option<Rewards>,
-    #[serde(default, skip_serializing_if = "Option::is_none")]
-    pub loaded_addresses: Option<UiLoadedAddresses>,
-    #[serde(default, skip_serializing_if = "Option::is_none")]
-    pub return_data: Option<UiTransactionReturnData>,
-    #[serde(default, skip_serializing_if = "Option::is_none")]
-    pub compute_units_consumed: Option<u64>,
+    pub inner_instructions: OptionSerializer<Vec<UiInnerInstructions>>,
+    pub log_messages: OptionSerializer<Vec<String>>,
+    pub pre_token_balances: OptionSerializer<Vec<UiTransactionTokenBalance>>,
+    pub post_token_balances: OptionSerializer<Vec<UiTransactionTokenBalance>>,
+    pub rewards: OptionSerializer<Rewards>,
```

Comment:
I think that all uses of `OptionSerializer` should include the serde attributes or else not be used to prevent the possibility that `OptionSerializer::Skip` is used in the future without adding the attributes.

Reply:
Fair. They'll be needed in #27601 anyway.